visualization/app/codeCharta/ui/edgeSettingsPanel/edgeSettingsPanel.component.html

```diff
@@ -27,7 +27,7 @@
 	</md-input-container>
 </md-slider-container>
 
-<md-slider-container class="md-block">
+<md-slider-container class="md-block" title="Scale height of the curves">
```

Comment:
```suggestion
<md-slider-container class="md-block" title="Adjust the curve height">
```

What curves are we speaking about in this case? This is not a blocker.

Reply:
We're talking about the height of the curves of the edges.
There was so many "ofs" that I left out the "edges" part